lib/features/in_app_review/in_app_review.dart

```diff
@@ -0,0 +1,23 @@
+import "dart:async";
+
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+
+import "../../hooks/use_effect_on_init.dart";
+import "business/in_app_rating_service_provider.dart";
+
+class InAppReviewWidget extends HookConsumerWidget {
+  const InAppReviewWidget({super.key, required this.child});
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final inAppRatingService = ref.read(inAppRatingServiceProvider);
```

Comment:
it's discouraged to ever use `ref.read` in the build method. We should prolly opt for `ref.watch` always to have the newest instance. In this case this shouldn't change at the moment but we don't know what future will bring us.

Reply:
Ofc, my mistake here. I wonder if some linter rule could be configured for this? 🤔

Reply:
yeah, actually it's on my board to set up riverpod linter made by the riverpod author